Makefile

```diff
@@ -17,6 +17,7 @@ install:
 
 build-services:
 	@ echo -e '$(BUILD_PRINT)Building the RDF Differ micro-services'
+	@ docker volume create rdf-differ-template
```

Comment:
I would not create the volume here; this shall be handled by another make target.
just delete this line and add the dependency to the make target below.